src/melee/gr/grcastle.h

```diff
@@ -8,15 +8,7 @@
 #include <dolphin/mtx.h>
 #include <baselib/forward.h>
 
-typedef struct unkCastle {
-    /* 0x000 */ u8 _pad[0x10C];
-    /* 0x10C */ HSD_GObj* x10C[5];
-    /* 0x120 */ u8 _pad2[0x134-0x120];
-    /* 0x134 */ u8 x134[5];
-} unkCastle;
-
-typedef void (*unkCastleCallback)(UNK_T, unkCastle*);
-typedef void (*unkCastleCallback2)(UNK_T, unkCastle*, Ground_GObj*);
+typedef struct unkCastle unkCastle;
```

Comment:
Other way around; this goes in `forward` and the struct goes here.

Reply:
lol yup, just realized